scripts/version_mapper.py

```diff
@@ -63,6 +68,9 @@ def update_switch_version_file(filename, new_version, cname, render_last):
         else:
             new_versions_list = current_versions_list
 
+        # Get the latest stable version
+        latest_stable_version = new_versions_list[0]
+
```

Comment:
Are we sure that element 0 is not the ``dev`` version? Just asking

Reply:
Yes, we are. When collecting available version numbers, the `dev` is ignored. When building the new `versions.json`, the `dev` is added on its own. Thus, `new_versions_list` only contains version numbers in descending order:

```pycon
>>> new_versions_list
["0.3", "0.2", "0.1"]
```

Reply:
Okay perfect then! Thanks @jorgepiloto for confirming it!